CircleOfFifths/src/org/puredata/android/fifths/CircleOfFifths.java

```diff
@@ -129,22 +129,17 @@ public void setTop(int top) {
 	@Override
 	public void onClick(View v) {
 		int newOption;
-		switch (v.getId()) {
-		case R.id.domdim:
+		int id = v.getId();
+		if (id == R.id.domdim) {
 			newOption = 2;
-			break;
-		case R.id.majmin:
+		} else if (id == R.id.majmin) {
 			newOption = 4;
-			break;
-		case R.id.sixth:
+		} else if (id == R.id.sixth) {
 			newOption = 6;
-			break;
-		case R.id.susp:
+		} else if (id == R.id.susp) {
 			newOption = 8;
-			break;
-		default:
+		} else {
 			newOption = 0;
-			break;
 		}
```

Comment:
This, and similar changes in this PR where `switch` were refactored to `if / else`, were required for the project to build. I was getting build errors, something that seems similar to [this](https://stackoverflow.com/questions/21005205/what-causes-constant-expression-required-errors-for-the-generated-r-id-xxx-val) issue.